### ISSUE TITLE
use 'when:' to begin block statements

### ISSUE THREAD

**Issue by miabbott** [projectatomic/atomic-host-tests#295]
@ashcrow pointed out that we can make blocks easier to read by using this syntax:

```
-  when:  something is true
   block:
     - task
     - task
```

Would be a nice cleanup to make across the repo

**Comment by ashcrow** [comment]
A bit of it is done in https://github.com/projectatomic/atomic-host-tests/pull/296/commits/f0f70dc815709e336cd90c9e2add8a1cc689436a for tasks.

**Note**: It's always good to double check syntax after with ``ansible-playbook --check-syntax path/to/main.yml`` 😸

**Comment by miabbott** [comment]
@kbidarkar is from the Satellite 6 team and is on QE Exchange with the Atomic Host QE team.  He's going to work on this issue.

**Comment by miabbott** [comment]
Closed via #311 and #313
